theme/less/homepage.less

```diff
@@ -0,0 +1,89 @@
+@import "_variables.less";
+@import "_body-wrapper.less";
+@import "_header.less";
+@import "_navigation.less";
+@import "_content.less";
+@import "_footer.less";
+
+body div
+{
+  grid-template-areas: 
+    "header header"
+    "main main"
+    "footer footer";
+}
+
+main article
+{
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  column-gap: 20px;
+}
+
+main article > *:first-child
+{
+  grid-column: 1 e("/") 3;
+}
+
+.box
+{
+  position: relative;
+  background-color: #f2f2f2;
+  border-radius: 8px;
+  border: 1px solid #dcdcdc;
+  padding: 10px 10px 60px 10px;
+  margin: 20px 0 0 0;
+  z-index: -1;
```

Comment:
This causes the links to be unclickable. Let's not use negative `z-index`.

Reply:
You can instead use positive `z-index` on the absolute `ul` element in navigation.